tests/components/fritzbox/__init__.py

```diff
@@ -150,7 +150,25 @@ class FritzDeviceLightMock(FritzEntityBaseMock):
     fw_version = "1.2.3"
     has_alarm = False
     has_powermeter = False
-    has_lightbulb = True
+    has_color = True
+    has_level = True
+    has_switch = False
+    has_temperature_sensor = False
+    has_thermostat = False
+    has_blind = False
+    level = 100
+    present = True
+    state = True
+
+
+class FritzDeviceNonColorLightMock(FritzEntityBaseMock):
```

Comment:
it is not needed to have a separate light mock, just override the affected properties during the test